database/src/lib.rs

```diff
@@ -812,7 +812,10 @@ pub enum BenchmarkRequestStatus {
     WaitingForArtifacts,
     ArtifactsReady,
     InProgress,
-    Completed { completed_at: DateTime<Utc> },
+    Completed {
+        completed_at: DateTime<Utc>,
+        duration_ms: u32,
```

Comment:
```suggestion
        duration: Duration,
```
In Rust we can just use a proper duration type.